generators/client/templates/angular/webpack/_webpack.prod.js

```diff
@@ -65,6 +65,19 @@ module.exports = webpackMerge(commonConfig({ env: ENV }), {
             ],
             exclude: ['node_modules/generator-jhipster']
         },
+        {
+            test: /\.ts$/,
+            use: [
+                {
+                    loader: 'ngc-webpack',
+                    options: {
+                        disable: false,
```

Comment:
Added to that for `resource override`, we have given empty string which again results in false. So we are not using resource override currently.